packages/web-client/app/components/card-pay/header/workflow-tracker/item/index.hbs

```diff
@@ -1,4 +1,5 @@
 
+{{!-- template-lint-disable no-invalid-interactive --}}
```

Comment:
I'm struggling to conform to accessibility rules with this design 🤔

To satisfy the rule where buttons shouldn't be nested under each other, I refactored the workflow item component to have two buttons that aren't nested - one being the description part and the other one contains the delete icon.

However, this still isn't quite alright. The linter rule says `Interaction added to non-interactive element` - this is due to `{{on 'mouseover']]` and `{{on 'mouseleave'}}` handlers set on the parent `div` in this component. If I change the `div` to an interactive element, then I'm faced with the original accessibility issue again, described above.

I think this particular design (hover to unveil the buttons) isn't too accessible from the get-go. Any ideas? Can we ignore this issue in this case, or maybe adjust the design spec?

Reply:
It's possible to make the delete icon button `display: none`, and control its visibility via CSS's `:hover` pseudo-class on the parent:

```
.workflow-tracker-item:hover .workflow-tracker-item__delete-icon {
  display: flex;
}
```

thus circumventing this problem. An additional step we can take is to use a screenreader-friendly implementation instead of `display: none;`, see `.boxel-sr-only`, Boxel's screenreader-only utility class.

Reply:
@jurgenwerk I think it's ok to ignore this issue for now. There are also issues around keyboard navigation - we don't show the delete button when. We can raise this in the design audit of Card Pay.

Reply:
Noted this in https://linear.app/cardstack/issue/CS-1745/visual-audit-of-card-pay-ui-on-production

Reply:
I see, thank you @Aierie!